packages/echo/src/types/skip.types.ts

```diff
@@ -1 +1,3 @@
-export type Skip = (payload: any) => boolean | Promise<boolean>;
+import { MaybePromise } from './util.types';
```

Comment:
Can we please rename this to [Awaitable](https://github.com/microsoft/TypeScript/issues/31394)? This is a public type so a good naming is crucial!

Reply:
Should we keep the MaybePromise as backward compatibility for a couple of versions?

Reply:
It's not needed. This is an alpha SDK, so far, and there are no guarantees for backward compatibility.